x-pack/test/detection_engine_api_integration/security_and_spaces/group2/create_endpoint_exceptions.ts

```diff
@@ -6,15 +6,16 @@
  */
 
 import { ToolingLog } from '@kbn/tooling-log';
-import expect from '@kbn/expect';
+import expect from 'expect';
```

Comment:
Are we moving away from our local fork of `expect`?

Reply:
'm not certain if it has been recently recommended as a best practice, but I noticed that the new tests are using `import expect from 'expect;'` instead of `@kbn/expect`. Additionally, I had to utilize `expect.arrayContaining` and `expect.objectContaining` to perform assertions on specific data parts rather than the entire data, resulting in reduced test flakiness.🤞